x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_configuration_form.tsx

```diff
@@ -56,21 +57,41 @@ export const ConnectorConfigurationForm = () => {
       component="form"
     >
       {localConfigView.map((configEntry) => {
-        const { depends_on: dependencies, key, label } = configEntry;
+        const {
+          default_value: defaultValue,
+          depends_on: dependencies,
+          key,
+          display,
+          label,
+          tooltip,
+        } = configEntry;
+        // toggle label goes next to the element, not in the row
         const hasDependencies = dependencies.length > 0;
+        const helpText = defaultValue
+          ? `If left empty, the default value ${defaultValue} will be used.`
```

Comment:
This needs to be internationalized

Reply:
I've added localisation. I did the steps according to documentation, let me know if something is wrong.